content/docs/overview/about.md

```diff
@@ -0,0 +1,95 @@
+---
+title: "About NuoDB Control Plane"
```

Comment:
Suggest removing the word "About". The term is redundant because it is already assumed the section is about or describes the "NuoDB Control Plane" 

We use to use the word "About" in the NuoDB doc page titling. It ended up an inconsistent formatting scheme. Some places used the term others did not. It was hard to understand why it was being used. We removed all known occurrences.

Reply:
Okay. I renamed the file as well.